xbmc/guilib/StereoscopicsManager.cpp

```diff
@@ -509,6 +509,8 @@ void CStereoscopicsManager::OnPlaybackStarted(void)
   STEREOSCOPIC_PLAYBACK_MODE playbackMode = (STEREOSCOPIC_PLAYBACK_MODE) CSettings::GetInstance().GetInt(CSettings::SETTING_VIDEOPLAYER_STEREOSCOPICPLAYBACKMODE);
   RENDER_STEREO_MODE mode = GetStereoMode();
 
+  g_infoManager.UpdateAVInfo();
```

Comment:
yeah, that's for sure the better aproach and ofc sufficient for this PR. No idea why I relied on GUI info for runtime decisions. Looking at it now that was really a bad idea.

However, @FernetMenta if you have a suggestion on how to handle/fix the resolution switch issue, I'm all ears. Also a new message/signal earlier in the process (like when all metadata has been collected/demuxed etc) would be good, because right now it can happen that ~1sec of the movie is already played before we ask for the prefered playback mode (2D vs 3D). Shall I open a thread in videoplayer dev section about it?